spec/support/loss_report_samples_helper.rb

```diff
@@ -0,0 +1,834 @@
+module Helpers
+  module LossReportSamplesHelper
+    def dif_id
```

Comment:
I would manage these ids with lets in the test files

Reply:
I originally defined it using `let`'s but I thought centralizing the definitions would eliminate the possibility that different collections could be tested in the loss_report_helper_spec.rb and the loss_report_spec.rb files.